package.json

```diff
@@ -88,8 +87,9 @@
     "jest": "^25.0.0",
     "jest-cli": "^25.0.0",
     "markdown-loader-jest": "^0.1.1",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
+    "prop-types": "^15.8.1",
+    "react": "^16.8.6 || ^17.0.0",
+    "react-dom": "^16.8.6 || ^17.0.0",
```

Comment:
We should only specify one version in the `devDependencies`, the latest version of React that Paragon will support. The range as you have it here should be reflected in the `peerDependencies`, as that is what tells consumers which versions of React this library supports.

As it is right now, it's saying Paragon relies only on React 18, yet installing... both (?) React 16 and 17.

Reply:
Yes, you are right
Made changes and continued working on the [new pull request](https://github.com/openedx/paragon/pull/1248)